src/omv/imlib/framebuffer.c

```diff
@@ -166,8 +167,32 @@ void framebuffer_update_jpeg_buffer() {
                     .size = OMV_JPEG_BUFFER_SIZE_MAX,
                     .pixels = jpeg_framebuffer->pixels
                 };
-                // Note: lower quality saves USB bandwidth and results in a faster IDE FPS.
-                bool overflow = jpeg_compress(src, &dst, jpeg_framebuffer->quality, false, JPEG_SUBSAMPLING_AUTO);
+                bool overflow = false;
+                if (0) {
+                #if OMV_RAW_PREVIEW_ENABLE
+                } else if (src->is_mutable &&
+                           image_size(src) <= OMV_JPEG_BUFFER_SIZE_MAX) {
+                    // Down-scale the frame (if necessary) and send the raw frame.
+                    dst.size = src->bpp;
+                    dst.pixfmt = src->pixfmt;
+                    if (src->w <= OMV_RAW_PREVIEW_WIDTH && src->h <= OMV_RAW_PREVIEW_HEIGHT) {
+                        memcpy(dst.pixels, src->pixels, image_size(src));
+                    } else {
+                        dst.w = OMV_RAW_PREVIEW_WIDTH;
+                        dst.h = OMV_RAW_PREVIEW_HEIGHT;
+                        if (src->w > src->h) {
+                            dst.h = fast_floorf(dst.h * (src->h / (float) src->w));
+                        } else if (src->h > src->w) {
+                            dst.w = fast_floorf(dst.w * (src->w / (float) src->h));
+                        }
+                        imlib_draw_image(&dst, src, 0, 0, 1.0f, 1.0f, NULL, -1, 255,
```

Comment:
Yeah, so, the 1.0f and 1.0f for x/y scale should definitely be the actual scaling values.

Not sure why this is passing. This invocation is telling the function to draw the image at the top left/right corner of the dst with no scaling.

I'll send a patch later. I can test with the RT1062 and switch it to raw image transport and update the IDE. I'll do that work next week maybe.

Reply:
The bilinear flag triggers the scaling, at least for the GPU. This works for me, feel free to update.